examples/worldview/src/features/stage/utils.js

```diff
@@ -0,0 +1,7 @@
+export const nearestEvenInt = num => 2 * Math.round(num / 2);
+
+export const nearestEvenFloat = num => {
+  num = num * 1000;
+  num = 2 * Math.round(num / 2);
+  return num / 1000;
+};
```

Comment:
What's your thought on making this function takes in decimal place rather than making it fixed at 1000?

ex)
`export const nearestEvenFloat = (num, decimalPlaces)`

Reply:
we can also do something like 
` factorTen = Math.pow(10, decimalPlaces)`

Reply:
Oh I like that. Just tested it out.